app/Collection.php

```diff
@@ -93,6 +93,11 @@ public function getResizedImage($resize)
         return $result_path;
     }
 
+    public function scopeContentImages($query)
```

Comment:
```suggestion
    public function getContentImages()
```
Scopes should be used in context of query building

Reply:
this is better now. let's keep it for now.
but for future - it's also possible to use "custom/virtual attributes"
if you name that function `getContentImagesAttribute()`, you can then access it with simple `$article->content_images` instead of `$article->getContentImages()`
to read more, check the docs -> https://laravel.com/docs/5.8/eloquent-mutators#defining-an-accessor